packages/react-components/src/components/PictureInPictureInPicture/PictureInPictureInPicture.tsx

```diff
@@ -48,16 +60,35 @@ export const _PictureInPictureInPicture = (props: _PictureInPictureInPictureProp
  */
 const PictureInPictureInPictureContainer = (props: {
   primaryView: ReactChild;
-  secondaryView: ReactChild;
+  secondaryView?: ReactChild;
```

Comment:
I don't understand why this is an optional prop now

Reply:
From PR "why" desc 😉 'Secondary tile being optional allows for us to have the local participant take up the whole tile when there are no remote participants'

Reply:
(It was bolted into this PR really from laziness as it was a pain to split out in the stack of PRs when I was splitting the big one up)